torch/csrc/autograd/functions/basic_ops.h

```diff
@@ -20,6 +20,10 @@ struct Error : public Function {
 
   virtual variable_list apply(const variable_list& inputs) override;
 
+  variable_list operator()(const variable_list& inputs) {
+    return apply(inputs);
+  }
```

Comment:
`operator()` isn't virtual (and we don't want it to be), so that's why this is a no-op.